controllers/fenceagentsremediation_controller.go

```diff
@@ -79,7 +80,17 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		r.Log.Error(err, "failed to get FAR CR")
 		return emptyResult, err
 	}
-	// TODO: Validate FAR CR name to nodeName. Run isNodeNameValid
+	// Validate FAR CR name to match a nodeName from the cluster
+	r.Log.Info("Check FAR CR's name")
+	if exist, err := farUtils.IsNodeNameValid(r.Client, req.Name); !exist {
```

Comment:
- nit:  `exists` doesn't match function name, `valid` or `isValid` would be better
- you rely on the bool value being false in case of an error. The bool value shouldn't matter though IMHO. Better first check for the error, then the bool.